src/scss/components/topnav/index.scss

```diff
@@ -109,14 +118,9 @@
       > hx-icon,
       p {
         color: $gray-950;
+        color: var(--hxTopNav-hxTopNavIconMenu-para-selected-color, $gray-950);
       }
     }
-
-    #topnav-menu:hover,                // deprecated
-    #topnavMenuId hx-menuitem:hover {
-      // deprecated
-      color: $gray-950 !important;
-    }
```

Comment:
Did you want to remove this?

Reply:
yes, we are using it in a different way

`hx-menuitem:not(.hxMenuKey):hover`